shell/platform/tizen/tizen_native_window.cc

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tizen_native_window.h"
+
+#include "flutter/shell/platform/tizen/logger.h"
+
+class TizenWl2Display {
+ public:
+  TizenWl2Display() {
+    if (!ecore_wl2_init()) {
+      LoggerE("Could not initialize ecore_wl2");
+      abort();
+      return;
```

Comment:
I think that If ecore_wl2_init fails, nothing can be done normally so abort is not bad solution.
Of course, since abort means an abnormal termination, I think it is far from safety.
if you have any suggestion please let' me know feel free. anyway I'll remove return

Reply:
We decided to keep the changes we initially proposed after ofline-discussion.
This case will eventually be handled by app-host.